src/pages/EditMonitor.vue

```diff
@@ -475,6 +512,17 @@ export default {
     mounted() {
         this.init();
 
+        this.$root.getMonitorList((res) => {
+            if (res.ok) {
+                Object.values(this.$root.monitorList).filter(monitor => monitor.id !== Number(this.$route.params.id)).map(monitor => {
+                    this.dependentMonitorsOptions.push({
+                        id: monitor.id,
+                        name: monitor.name,
+                    });
+                });
+            }
+        });
```

Comment:
@Saibamen I think I will adjust this function to load all monitors from the database, except those that depend on that monitor.

What do you think about it?

Reply:
This will prevent monitor y from relying on monitor x if monitor x is already dependent on monitor y.

Reply:
Looks good

Reply:
@Saibamen should be OK now 🙂